### ISSUE TITLE
ClassDefNot found exception requiring bosquet

### ISSUE THREAD

**Issue by behrica** [zmedelis/bosquet#29]
using 0.3.6 I get a exception requiring the 'bosquet.generator" namespace:

```
(require '[bosquet.generator :as bg])
Execution error (FileNotFoundException) at selmer.tags/loading (tags.clj:1).
Could not locate selmer/node__init.class, selmer/node.clj or selmer/node.cljc on classpath.
```
deps.edn:
```
{:deps {io.github.zmedelis/bosquet {:mvn/version "0.3.6"}}}
```
on Linux

**Comment by behrica** [comment]
Strangely it works when inside the bosquet source code (on tag v0.3.6)
(The above is in a project which is using bosquet)

**Comment by behrica** [comment]
I think the issue is that the git dependeny to selmer is not "transitive" resolved when using a mvn dependency to bosquet.

**Comment by behrica** [comment]
Indeed, with this deps.edn it work:

```
{:deps {io.github.zmedelis/bosquet {:mvn/version "0.3.6"}
            zmedelis/Selmer            {:git/url "https://github.com/zmedelis/Selmer"
                                                    :git/sha "88fc2b51817e1ad67f0458c75857222a72db4945"}}}
```

**Comment by zmedelis** [comment]
> I think the issue is that the git dependeny to selmer is not "transitive" resolved when using a mvn dependency to bosquet.

Yes POM can't deal with that. I will make my Selmer fork release and refer to that

**Comment by zmedelis** [comment]
The commit addresses this. Not closing the issue till I make a new Bosqute release with a few more fixes to the critical issues you reported.

**Comment by zmedelis** [comment]
Addressed in the v0.3.7 release.
